forms-shared/src/slovensko-sk/generateXml.ts

```diff
@@ -18,8 +20,8 @@ function getSlovenskoSkXmlBase(formDefinition: FormDefinitionSlovenskoSk, body:
   }
 }
 
-export function getEmptySlovenskoSkXml(formDefinition: FormDefinitionSlovenskoSk) {
-  const xml = getSlovenskoSkXmlBase(formDefinition, {
+export function getEmptySlovenskoSkXmlObject(formDefinition: FormDefinitionSlovenskoSk) {
```

Comment:
would add jsdoc detailing the output can be used with buildSlovenskoSkXml function to generate a valid, empty response

similar in generateSlovenskoSkXmlObject, that it creates a valid representation which is supposed to be used with the build function

Reply:
Added comment